go/test/fuzzing/vttablet_fuzzer.go

```diff
@@ -291,17 +291,6 @@ func (fs *fuzzStore) callExecuteFetchAsApp() error {
 	return nil
 }
 
-// callInitMaster implements a wrapper
-// for fuzzing InitMaster
-func (fs *fuzzStore) callInitMaster() error {
-	tablet, err := fs.getTablet()
-	if err != nil {
-		return err
-	}
-	_, _ = fs.client.InitMaster(context.Background(), tablet, false)
-	return nil
-}
```

Comment:
We shouldn't rename this? I don't see an existing `callInitPrimary()` function. In the vttablet_fuzzer we seem to be removing functions rather than renaming/replacing. Is that intentional and correct? Otherwise the changes LGTM.

Reply:
yeah we should rename if there isn't already one. I'll fix it.

Reply:
Renamed the ones where the new RPCs weren't already included.